README.md

```diff
@@ -1,10 +1,41 @@
-# Planner
-As part of the Learn IT Girl I am developping a Java desktop application. The application should be a planner of day by day tasks and important events. The application will offer a calendar like interface, where users can view tasks for each day and also create tasks.
-The planner offers the following functionalities:
-- monthly view of the calendar and weekly view
-- prioritization of tasks
-- creating one time or recurring events
-- 15 minutes, 10 minutes, “starting now” reminders for tasks. 
+Planner
+As part of the Learn IT Girl I was developping a Java desktop application. I used Java 8. 
+The application is a planner of day by day tasks and important events. The Planner offers a calendar like interface, where users can view tasks for each day and also create tasks. 
+Planner uses also the MySQL database. The lists of the users and the tasks are keeping in the database. 
 
+The planner offers the following functionalities:
+- monthly view of the calendar,
+- creating one time or recurring events (repeatable monthly and yearly),
+- 15 minutes, 10 minutes, “starting now” reminders for tasks.
 The planner will help you organize your time and your family's time.
 
+The instruction of database in Planner (for Windows):
+- you need to download the MySQL Installer from: https://dev.mysql.com/downloads/installer/
+- you need to install MySQL Server, MySQL Connector/J and MySQL Workbench. During the instalation program can ask you to download additional packages.
```

Comment:
Here  I would add a ',': 'During the installation, the program can ask you to download additional packages.'